src/pages/dashboard.vue

```diff
@@ -85,7 +85,7 @@ const containRole = (roleType) => {
 const fetchBusinessDetailsWithDelay = async (identifier: string) => {
   try {
     const slimBusiness = await business.getBusinessDetails(identifier, undefined, true)
-    const lastModifiedDate = slimBusiness.lastModified ? apiToDate(slimBusiness.lastModified) : null
+    const lastModifiedDate = slimBusiness?.lastModified ? apiToDate(slimBusiness.lastModified) : null
```

Comment:
Now I'm rethinking my previous comment...

If the business response is "No information on temp registrations." instead of a business object then we never detect a change, right? In this case, it makes no sense to keep polling/retrying forever. See if you can disable that.

I do think the conditional chaining is required above to avoid an exception.

Please create a new ticket to find a solution for identifying changes to a T business after the dashboard has already loaded.

Reply:
Wonderful, thanks!
Resolving.